init-service/src/test/java/uk/ac/ox/ndph/mts/init_service/service/PractitionerServiceTest.java

```diff
@@ -14,13 +14,13 @@
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.test.util.ReflectionTestUtils;
 import org.springframework.web.reactive.function.client.WebClient;
-import uk.ac.ox.ndph.mts.init_service.configuration.WebClientConfig;
 import uk.ac.ox.ndph.mts.init_service.config.AzureTokenService;
 import uk.ac.ox.ndph.mts.init_service.exception.DependentServiceException;
 import uk.ac.ox.ndph.mts.init_service.model.IDResponse;
 import uk.ac.ox.ndph.mts.init_service.model.Practitioner;
 import uk.ac.ox.ndph.mts.init_service.model.PractitionerUserAccount;
 import uk.ac.ox.ndph.mts.init_service.model.RoleAssignment;
+import uk.ac.ox.ndph.mts.roleserviceclient.configuration.WebClientConfig;
```

Comment:
Can we revert to using uk.ac.ox.ndph.mts.init_service.configuration.WebClientConfig, make changes to match init_service WebClientConfig to roleserviceclient WebClientConfig

Reply:
In the next PR  uk.ac.ox.ndph.mts.init_service.configuration.WebClientConfig is going to be removed any way...

Reply:
Ok that's fine.